docs/api/session.md

```diff
@@ -553,6 +553,46 @@ session.fromPartition('some-partition').setPermissionCheckHandler((webContents,
 })
 ```
 
+#### `ses.setMediaRequestHandler(handler)`
+
+* `handler` Function | null
+  * `request` Object
+    * `frame` [WebFrameMain](web-frame-main.md) - frame that is requesting access to media
+    * `type` String - Type of request. Can be 'deviceAccess', 'deviceUpdate', 'generateStream' or 'openDevicePepperOnly'.
+    * `audioType` String - can be 'deviceAudioCapture', 'displayAudioCapture' or 'noService'.
```

Comment:
Could these be 'device', 'display', and `null`? The 'AudioCapture' is redundant since the key is 'audioType'.